pkg/rules/signature/signature.go

```diff
@@ -49,7 +49,8 @@ func getSignatures(target string, partialEval bool, rulesDir string, rules []str
 	} else {
 		for _, s := range sigs {
 			for _, r := range rules {
-				if m, err := s.GetMetadata(); err == nil && m.ID == r {
+				if m, err := s.GetMetadata(); err == nil &&
+					(m.ID == r || m.EventName == r) {
```

Comment:
Isn't ID check sufficient? Have ID and EventName the same value?

Reply:
hmm, how would we do it? The ids are `TRC-1` for example, events are more meaningful, like `anti_debugging_ptraceme`, making eventName `TRC-1` have no meaning, and changing ID for eventName will break the compatibility of the signatures.

Reply:
I got it. Agreeing with your implementation.

Reply:
I don't find it very critical myself, but my two cents on the issue is that i'm not sure if we should allow selecting signatures through their event names in the same context, but again, not too attached to it either way.